workflows/utilities/wf_organism_parameters.wdl

```diff
@@ -0,0 +1,162 @@
+version 1.0
+
+workflow organism_parameters {
```

Comment:
Seems like we can also apply this to the [augur defaults tasks](https://github.com/theiagen/public_health_bioinformatics/blob/acee8134fd75ea8dd734b322a3bbc9b75728212d/workflows/phylogenetics/wf_augur.wdl#L43) in a future PR

Reply:
Definitely agree. This could be used in a lot of different areas, and could reduce the contents needed in our input JSONs as well.